Lombiq.VueJs/package.json

```diff
@@ -14,23 +12,31 @@
   },
   "devDependencies": {
     "@vue/compiler-sfc": "^3.2.29",
+    "eslint": "^8.17.0",
     "fancy-log": "^2.0.0",
```

Comment:
Why do we include minor version updates here but use fixed versions for the the rest of the packages?

Reply:
No idea.

Reply:
Let's use the same `^` there too, then.